### ISSUE TITLE
Git error updating to 1.6.7rc7

### ISSUE THREAD

**Issue by kalbermattenm** [camptocamp/c2cgeoportal#1955]
I just tried to update to c2cgeoportal 1.6.7rc7 and running this command, I get the error hereunder:

```
 .build/venv/Scripts/c2ctool upgrade --windows michael.mk 1.6.7rc7 --step 1
```

I think that this <a href="https://github.com/camptocamp/c2cgeoportal/blob/1.6/c2cgeoportal/scripts/c2ctool.py#L305" target=_blank>line</a> is guilty for the error...

Regarding "update", the Git commands could be: `update-index`, `update-ref` or `update-server-info`

```shell
git : 'update' n'est pas une commande git. Voir 'git --help'.

Vouliez-vous dire cela ?
        update-ref
Traceback (most recent call last):
  File "C:\Python27\Lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\Lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\git_projects\sitn_c2cgeoportal\.build\venv\Scripts\c2ctool.exe\__main__.py", line 9, in <module>
  File "c:\git_projects\sitn_c2cgeoportal\.build\venv\lib\site-packages\c2cgeoportal\scripts\c2ctool.py", line 106, in main
    c2ctool.upgrade()
  File "c:\git_projects\sitn_c2cgeoportal\.build\venv\lib\site-packages\c2cgeoportal\scripts\c2ctool.py", line 240, in upgrade
    self.step1()
  File "c:\git_projects\sitn_c2cgeoportal\.build\venv\lib\site-packages\c2cgeoportal\scripts\c2ctool.py", line 305, in step1
    check_call(["git", "update", "origin", "--prune"])
  File "C:\Python27\Lib\subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'update', 'origin', '--prune']' returned non-zero exit status 1
```

**Comment by gnerred** [comment]
+1 I get the same error

**Comment by sbrunner** [comment]
Thanks, will be fixed with:  #1931

**Comment by kalbermattenm** [comment]
PR was merged, branch update, so this is fixed, closing.
